cypress/integration/basic/role_assignment_made.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Test that role assignments can be made', () => {
```

Comment:
Update describe() and it() language as explained here: https://github.com/dssit/roles-management/pull/349